compiler/rustc_session/src/options.rs

```diff
@@ -1647,6 +1665,8 @@ options! {
         "instrument control-flow architecture protection"),
     check_cfg_all_expected: bool = (false, parse_bool, [UNTRACKED],
         "show all expected values in check-cfg diagnostics (default: no)"),
+    checksum_hash_algorithm: Option<SourceFileHashAlgorithm> = (None, parse_cargo_src_file_hash, [TRACKED],
+        "hash algorithm of source files used to check freshness in cargo (`sha256`)"),
```

Comment:
```suggestion
        "hash algorithm of source files used to check freshness in cargo (`blake3`)"),
```

Reply:
Fixed, though actually both blake3 and sha256 are supported here.